css/input.css

```diff
@@ -0,0 +1,112 @@
+#armor {
```

Comment:
the armor and endurance charges look kinda funny with their vertical alignment now no?